index.rst

```diff
@@ -6,24 +6,219 @@
 
 .. TODO: Delete the note below before merging new content to the main branch.
 
-.. note::
-
-   **This technote is a work-in-progress.**
-
 Abstract
 ========
 
-This is a technote for the description of results from Individual Hardpoint Breakaway Test. 
+This technote describes the analysis and results derived from individual hard point breakaway tests conducted with the M1M3 surrogate.
+The notebook is located within `notebooks_vandv <https://github.com/lsst-sitcom/notebooks_vandv/blob/tickets/SITCOM-838/notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-838_Anaysis.ipynb>`_ GitHub repository.
+With all the steps guarded with reasonable timeouts, so problems are detected if hard point cannot travel to reach low or high limit switches, etc.
+**If this test shows that the hard points do not work properly at the limits, this could be one of the blockers for the installation of M1M3 until it is solved.** 
+
+
+Hardpoint Breakaway Test
+========================
+
+The active support system of the M1M3 includes six axial hard point actutators in the hexapod configuration. :cite:`2018SPIE10700E..3GD`
+These hard point actuators should minimize forces during slews at any TMA position and be kept under the breakaway limit. 
+The breakaway limits for each hard points should happen in the range of -4420 to -3456 N for retraction and 2981 to 3959 N for extension.
+The followings steps are performed during the individual hard point breakaway test.
+
+1. Move hard point in negative (increasing tension) direction until either a low limit switch is actuated
```

Comment:
I don't think that we have a "limit switch." We might want to clarify a bit more what is a "break away" in this context. I would review the "breakaway" definition with Doug, but I believe it is when the Hard Point Break Away system receives a force so high that it "gives up."

Reply:
I already forwarded the response from Petr to you but those steps were from Petr. But let's put it on the To-do list to check with others. 
------

_1. Move hardpoint in negative (increasing tension) direction until either a low limit switch is actuated
2. Move hardpoint in positive (increasing compression) direction till the high limit switch is triggered
3. Move hardpoint downwards (increasing tension) until the low limit switch is hit
4. Move hardpoint back to reference position
5. Wait for the hardpoint to reach the reference position__